dashboard/app/views/parent_mailer/parent_email_added_to_student_account.haml

```diff
@@ -1,7 +1,8 @@
 %p
-  Welcome to Code.org! You have successfully linked your email to your child’s account.
-  We're thrilled that you have taken the next step in supporting #{@student.name} in their learning and exploration of
-  computer science. If you are not #{@student.name}’s parent, please reply to this email or contact support@code.org.
+  Welcome to Code.org! You have successfully linked your email to the account of your child,
+  %strong>= " #{@student.name}"
+  \. We're thrilled that you have taken the next step in supporting your child in their learning and exploration of
+  computer science.  If you are not the parent or guardian for this account, please reply to this email or contact support@code.org.
```

Comment:
will `support@code.org` automatically become a mailto link?

Reply:
It does for me. I don't know if this depends on the mail client though.